documentation/src/main/doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -26,7 +26,7 @@ Creates a Maven project, and include the following dependency in your pom.xml:
 <dependency>
   <groupId>io.smallrye.config</groupId>
   <artifactId>smallrye-config</artifactId>
-  <version>1.7.0</version>
+  <version>1.8.0</version>
```

Comment:
this version should be an attribute. Can you open an issue about that?

Reply:
I can just quickly fix. Please hold on.

Reply:
Fixed.